insights/specs/default.py

```diff
@@ -336,6 +336,7 @@ class DefaultSpecs(Specs):
     ls_etc = simple_command("/bin/ls -lan {0}".format(' '.join(etc_and_sub_dirs)), keep_rc=True)
     ls_etc_ssh = simple_command("/bin/ls -lanL /etc/ssh")
     ls_ipa_idoverride_memberof = simple_command("/bin/ls -lan /usr/share/ipa/ui/js/plugins/idoverride-memberof")
+    ls_krb5_sssd = simple_command("/bin/ls -lan /var/lib/sss/pubconf/krb5.include.d")
```

Comment:
@rasrivas-redhat please add the file spec to insights/specs/insights_archive.py so that the spec collected via JSON can be accessed from the archive.  Use the `mangle` command to determine what file path to use.

Reply:
@bfahr - Thanks for noticing it, added related spec to insights/specs/insights_archive.py